contents/node_operator.rst

```diff
@@ -412,6 +412,101 @@ If the external node is running on IP 12.34.56.78 port 8545, the command would b
 
 You can also use ``https``. But keep in mind that Swarm *does not validate the certificate*.
 
+Connect to the SWAP-enabled testnet
+===================================
+The Swarm project now runs a SWAP-enabled (incentivized) testnet. It uses the same binary as the normal network node, but has incentivization switched on. 
+
+.. important::
+
+  The testnet is highly experimental and may be updated continuously until we release a mainnet version. All tokens used on this testnet are fictitious. Do NOT use this testnet with real value tokens. Funds can be lost.
+
+ 
+The first public incentives-enabled Swarm network runs on the ropsten network. 
+It runs with network id **5**
+
+Prerequisites
+-----------------
+* Create an account on **ropsten**
+* Fund your newly created account with test coins
+
+Network setup
+-------------
+We provide on the public swap-enabled network currently 30 nodes in a kubernetes cluster. 
+
+Bootnodes
+--------------
+
+The bootnodes for the public swap-enabled Swarm cluster run in a separate VMs. 
+The addresses are 
+
+   .. code-block:: none
+
+      enode://7f4d606c91d50d91fd09cb44f8b3d8033f1ca87e977a881e91d77ff6af98b6a52245ba9aeba13a39024ae8bdf3afa421fd018571ae37928c065d7a62503f17a6@3.122.203.99:40301
+      enode://7f4d606c91d50d91fd09cb44f8b3d8033f1ca87e977a881e91d77ff6af98b6a52245ba9aeba13a39024ae8bdf3afa421fd018571ae37928c065d7a62503f17a6@3.122.203.99:40301
+
+Add these addresses with the `--bootnodes` when starting your node.
```

Comment:
It would be great if we could have them hardcoded!

Reply:
Yes, today, non-swap nodes are default, and those are the ones hardcoded.

When we move to mainnet and swap enabled should be default, then those should be hardcoded.

Currently I think this is correct.